Sources/Action/Action.swift

```diff
@@ -122,23 +107,20 @@ public final class Action<Input, Element> {
             inputs.onNext(value)
         }
 
-        let execution = executionObservables
-            .take(1)
-            .flatMap { $0 }
-            .catchError { throw ActionError.underlyingError($0) }
-
-        let notEnabledError = inputs
-            .takeUntil(executionObservables)
-            .withLatestFrom(enabled)
-            .flatMap { $0 ? Observable<Element>.empty() : Observable.error(ActionError.notEnabled) }
-
-        let subject = ReplaySubject<Element>.createUnbounded()
-        Observable
-            .of(execution, notEnabledError)
-            .merge()
-            .subscribe(subject)
-            .addDisposableTo(disposeBag)
-
-        return subject.asObservable()
+		let subject = ReplaySubject<Element>.createUnbounded()
+		
+		let work = executionObservables
+			.map { $0.catchError { throw ActionError.underlyingError($0) } }
```

Comment:
If you look at the previous code, it was doing exactly this -- propagate errors to the execution observable returned by the `execute()` function, both for errors produced by the execution, and for the `notEnabled` error. I kept the same behaviour.